static/app/views/issueDetails/streamline/eventGraph.tsx

```diff
@@ -233,7 +232,7 @@ export function EventGraph({group, event, ...styleProps}: EventGraphProps) {
       seriesData.push(releaseSeries as BarChartSeries);
     }
 
-    if (flagSeries.markLine && hasFeatureFlagFeature) {
+    if (flagSeries.markLine && flagSeries.type === 'line') {
```

Comment:
where did this condition come from? the image in the description looks like it's a bar chart...

Reply:
the flag series (the lines showing the feature flag changes) is a line chart. the bar chart part is the event/user graph

Reply:
see comment here https://github.com/getsentry/sentry/blob/1cabfbca746112f1dee03b51e61ea1136d554a75/static/app/views/issueDetails/streamline/hooks/useFlagSeries.tsx#L83

Reply:
oh, i see the `line` thing is unset when we return early with:

```
return {
  seriesName: t('Feature Flags'),
  markLine: {},
  data: [],
};
```

Reply:
yup, so if the type isn't set as `line` we know there's no flag chart data. so we don't want to show the legend